### PR TITLE
allow to override runtimesManifest

### LINKED ISSUE #2742: Add ability to override runtimesManifest in ansible
There is no easy way to override the runtimesManifest via target environment in ansible.

The use case is if I want to deploy whisk with only supporting a specific runtime kinds, or add additional custom ones like a new kind `nodejs-acme:8.5` 😉

### PR DESCRIPTION
**Pull request by csantanapr**
Closes #2742

### CODE REVIEW

File: ansible/group_vars/all [hunk 1]
```diff
@@ -92,6 +92,8 @@ runtimesManifest:
   blackboxes:
     - name: "dockerskeleton"
 
+runtimesManifest: "{{ runtimes_Manifest | default(runtimesManifestDefault) }}"
```
Comment (rabbah) [inline ansible/group_vars/all]: Why the capital M after underscore?

Comment (csantanapr) [reply ansible/group_vars/all]: I like MegaMan 📣

File: ansible/group_vars/all [hunk 2]
```diff
@@ -30,7 +30,7 @@ whisk:
 #   runtimes: set of language runtime families grouped by language (e.g., nodejs, python)
 #   blackboxes: list of pre-populated docker action images as "name" with optional "prefix" and "tag"
 #
-runtimesManifest:
+runtimesManifestDefault:
```
Comment (markusthoemmes) [inline ansible/group_vars/all]: This change is I think not necessary. We can leave the name as is, the distinction is made via snake_case vs. pascalCase.

Comment (csantanapr) [reply ansible/group_vars/all]: I'm confuse, I need a new name to hold the default values, take a look below where I define `runtimesManifest`

Comment (markusthoemmes) [reply ansible/group_vars/all]: Oh, right. My bad I guess it was a bit early this morning.

Comment (csantanapr) [reply ansible/group_vars/all]: Can you approve now?

**Comment by csantanapr** [comment]
PG1 2059 🔵

**Pull request merged**